vnext/PropertySheets/External/Microsoft.ReactNative.Common.targets

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+  Copyright (c) Microsoft Corporation.
+  Licensed under the MIT License.
+
+  This file will be consumed by ALL app and module projects (Desktop & UWP,
+  both inside and outside of this repo) that build on top of
+  Microsoft.ReactNative.
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+   
+   <Target Name="ReactNativeWindowsEnsureDir" BeforeTargets="PrepareForBuild">
+     <Error Condition="'$(ReactNativeWindowsDir)' == ''" Text="'ReactNativeWindowsDir' should be set to the resolved location of the `react-native-windows` package. See https://github.com/microsoft/react-native-windows/blob/446e78b7a8a4601513b96dee53672520015623dc/vnext/template/cpp-lib/proj/MyLib.vcxproj#L23 for an example." />
+  </Target>
```

Comment:
I don't think this will actually work. MSBuild no longer allows Error outside of target, but putting this in a target means it would be executed after initial property evaluation, which is where the failure happens.

Reply:
Looks like `InitialTargets` can be used for property validation. https://github.com/dotnet/msbuild/issues/3858

Reply:
Nope, that's still too late. We fail to load the project at all... Have a jank way to address this...